pkgs/development/compilers/zig/0.10.nix

```diff
@@ -2,53 +2,23 @@
 , stdenv
 , fetchFromGitHub
 , cmake
-, coreutils
 , llvmPackages
 , libxml2
 , zlib
-}:
+, coreutils
```

Comment:
??

Reply:
the inputs are explicitly listed so I can pass it to `generic.nix` with `import` instead of `callPackage`, to make sure that overrides work correctly

Reply:
> Why change the order?

I'm following the documented convention

> [Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.](https://nixos.org/manual/nixpkgs/unstable/#sec-syntax)

Reply:
Ok, I will propose a change on this.